src/guest_memory.rs

```diff
@@ -95,30 +99,6 @@ impl From<volatile_memory::Error> for Error {
 /// Result of guest memory operations.
 pub type Result<T> = std::result::Result<T, Error>;
 
-impl std::error::Error for Error {}
-
-impl Display for Error {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "Guest memory error: ")?;
```

Comment:
This prefix seems to be missing with the `thiserror` impls

Reply:
This is unfortunate design, but you are right, added the prefix to be identical.